off-chain/src/HandleActions.purs

```diff
@@ -147,3 +179,13 @@ getFinalizedTransactionHash :: FinalizedTransaction -> Contract TransactionHash
 getFinalizedTransactionHash fTx = do
   liftEffect $ Ctl.Internal.Hashing.transactionHash <$>
     Ctl.Internal.Serialization.convertTransaction (unwrap fTx)
+
+findOwnOutputs :: ValidatorHash -> UtxoMap -> Contract UtxoMap
```

Comment:
Looks like it duplicates one from `Seath.Test.Examples.Utils`. Maybe move Utils module to `src`?

Reply:
yes, that's the intention, I don't think src should depend on the test code.